File: Data Structures & Algorithms/duplicate-integer/submission-1.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:

        # num_set = set(nums)

        # if (len(num_set) != len(nums)):
        #     return True
        
        # return False
         
        hashm = {}

        for i in nums:
        
            if i not in hashm:
                hashm[i] = 1
            else:
                return True

        return False

File: Data Structures & Algorithms/jump-game/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    canJump(nums) {

        let goal = nums.length - 1;
        
        for(let i = nums.length; i >= 0;  i--){
            if(i + nums[i] >= goal){
                goal = i
            }
        }
        if(goal === 0){
            return true
        }else{
            return false
        }
    }
}
